packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/examples/troubleshooting/api-component-not-working.bash

```diff
@@ -0,0 +1 @@
+`You're attempting to use the `[ScannerAPI](/docs/api/pos-extensions/ui-extensions-reference/api/scanner)`, but it doesn't work. You've done everything according to the documentation.`
```

Comment:
Do these still render in a nice way if you use `.txt` instead? This isn't a bash script